api/cluster/EnvironmentRestHandler.go

```diff
@@ -288,7 +288,18 @@ func (impl EnvironmentRestHandlerImpl) GetEnvironmentListForAutocomplete(w http.
 
 		result := impl.enforcer.EnforceByEmailInBatch(emailId, casbin.ResourceGlobalEnvironment, casbin.ActionGet, envIdentifierList)
 		for _, item := range environments {
-			if hasAccess := result[strings.ToLower(item.EnvironmentIdentifier)]; hasAccess {
+
+			var hasAccess bool
+
+			if item.EnvironmentIdentifier != item.ClusterName+"__"+item.Namespace {
+				// fix for futuristic case
+				rbacObject2 := item.ClusterName + "__" + item.Namespace
```

Comment:
you are anyhow creating new var for this value, instead of fmt you can declare this before if condition and use this only everywhere